.github/DISCUSSION_TEMPLATE/component-proposals.yml

```
@@ -0,0 +1,93 @@
+title: "Component proposal: [Component name]"
+labels: ["Type: New component proposal"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for sharing your new component suggestion for the U.S. Web Design System (USWDS).
+        </p><p>
+        We ask for a lot of detail in this form, but it’s okay to start with basic information. Only the fields marked with an asterisk (*) are required at this time. You can work with the community to add more detail after you start this discussion. It helps if you link to supporting evidence and research wherever you can.
+        </p><p>
+        <a href="https://github.com/uswds/uswds/discussions/5764">Learn more about the component proposal process</a>.
```

Just curious, markdown in this field doesn't work? There are line break characters to keep the spacing - [YAML Docs](https://yaml.org/spec/1.2.2/#54-line-break-characters).

```suggestion
        Thanks for sharing your new component suggestion for the U.S. Web Design System (USWDS).

        We ask for a lot of detail in this form, but it’s okay to start with basic information. Only the fields marked with an asterisk (*) are required at this time. You can work with the community to add more detail after you start this discussion. It helps if you link to supporting evidence and research wherever you can.
       
[Learn more about the component proposal process](https://github.com/uswds/uswds/discussions/5764).
```

---

Thanks for calling this out. I swear those line breaks were causing problems earlier, but it must have been something else I was doing. I tested it out again in the demo repo and it worked. 🤷 Appreciate the comment!

See the changes in the demo repo here:
- [Updated Demo file](https://github.com/amyleadem/uswds-proposals/blob/main/.github/DISCUSSION_TEMPLATE/component-proposals-friendly-template.yml)
- [Updated Demo preview](https://github.com/amyleadem/uswds-proposals/discussions/new?category=component-proposals-friendly-template)

Removed the html in this PR in [babcc21](https://github.com/uswds/uswds/pull/5785/commits/babcc21f894df39b6dc0606e4f8b2b23d72ace75)